cmd/team_test.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/astronomer/astro-cli/houston"
+	mocks "github.com/astronomer/astro-cli/houston/mocks"
+	testUtil "github.com/astronomer/astro-cli/pkg/testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestNewGetTeamCmd(t *testing.T) {
+	testUtil.InitTestConfig()
+
+	team := &houston.Team{
+		Name: "Everyone",
+		ID:   "blah-id",
+	}
+	appConfig := &houston.AppConfig{
+		NfsMountDagDeployment: false,
+	}
+
+	api := new(mocks.ClientInterface)
+	api.On("GetTeam", mock.Anything).Return(team, nil)
+	api.On("GetAppConfig").Return(appConfig, nil)
+
+	output, err := executeCommandC(api, "team", "get", "test-id")
+	assert.NoError(t, err)
+	assert.Contains(t, output, "")
```

Comment:
nitpicking: Ideally we should also assert the mocks which are defined in the test as well, just to be sure that whatever mocks we have defined in the unit test are actually being used.
```suggestion
	assert.Contains(t, output, "")
	api.AssertExpectations(t)
```
In this case `api.AssertExpectations(t)` will make sure `GetTeam` & `GetAppConfig` mock method are actually being called while executing this test case.